jira/src/main/kotlin/gropius/sync/jira/JiraSync.kt

```diff
@@ -37,9 +42,14 @@ final class JiraSync(
     val imsConfigManager: IMSConfigManager,
     collectedSyncInfo: CollectedSyncInfo,
     val loadBalancedDataFetcher: LoadBalancedDataFetcher = LoadBalancedDataFetcher(),
-    val issueDataService: IssueDataService
+    val issueDataService: IssueDataService,
+    val syncStatusService: SyncStatusService
 ) : AbstractSync(collectedSyncInfo) {
 
+    companion object {
+        val jqlFormatter = DateTimeFormatter.ofPattern("\"yyyy-MM-dd HH:mm\"")
```

Comment:
docs
also, should be in SCREAMING_SNAKE_CASE according to Kotlin style guids

Reply:
8cf3ebd07ce301810b0dc5edad61e48820d05d96